scanner/config.go

```diff
@@ -91,7 +93,15 @@ func Config(searchDir string) models.ScanResultModel {
 	// Setup
 	currentDir, err := os.Getwd()
 	if err != nil {
-		result.AddError("general", fmt.Sprintf("Failed to expand current directory path, error: %s", err))
+		result.AddErrorWithRecommendation("general", models.ErrorWithRecommendations{
+			Error: fmt.Sprintf("Failed to expand current directory path, error: %s", err),
```

Comment:
`Failed to expand current directory path, error: %s` -> `Failed to expand current directory path: %s` and apply this to the above 2 places